src/control-plane/backend/lambda/api/service/project.ts

```diff
@@ -146,6 +146,10 @@ export class ProjectServ {
           AwsAccountId: process.env.AWS_ACCOUNT_ID,
           DashboardId: dashboardId,
         });
+        await quickSightClient.deleteAnalysis({
```

Comment:
How about the data set for this dashboard?

Reply:
delete datasets of dashboard